deploy/olm-catalog/kabanero-operator/0.0.1/kabanero-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,209 @@
+    mediatype: image/png
+  maintainers:
+  - name: Kyle Schlosser
+    email: kschloss@us.ibm.com
+  - name: Tim Kaczynski
+    email: kaczynsk@us.ibm.com
+  - name: Ed Mezarina
+    email: mezarina@us.ibm.com
+  - name: Tim Spewak
+    email: spewak@us.ibm.com
+  - name: Mike Ginnick
+    email: ginnick@us.ibm.com
+  links:
+  - name: Kabanero.io
+    url: https://kabanero.io
+  - name: Kabanero Operator
+    url: https://github.com/kabanero-io/kabanero-operator
+  keywords:
+  - 'Microservice'
+  - 'Knative'
+  - 'Istio'
+  - 'Codewind'
+  - 'Appsody'
+  - 'Tekton'
+  customresourcedefinitions:
+    owned:
+    - description: Kabanero Platform Operator 
+      displayName: Kabanero Operator
+      group: kabanero.io
+      kind: Kabanero
+      name: kabaneros.kabanero.io
+      resources:
+      - kind: ConfigMap
```

Comment:
I don't think this needs to be fixed now, but the resources should only contain the things that the operator creates.  I don't know off the top of my head what it is, but, should probably be just the three dependent CR instances that we create (pipeline, kserve, kevent).  If there are others that the operator directly creates, those can be here too.